R/assertions.R

```diff
@@ -489,12 +495,12 @@ assert_filter_cond <- function(arg, optional = FALSE) {
 
   provided <- !is_missing(arg)
   if (provided && !(is_call(arg) || is_logical(arg))) {
-    err_msg <- sprintf(
-      "`%s` must be a filter condition but is %s",
-      arg_name(substitute(arg)),
-      what_is_it(arg)
+    cli_abort(
+      message = message %||%
+        "Argument {.arg {arg_name}} must be a filter condition, but is {.obj_type_friendly {arg}}",
+      class = c(class, "assert-admiraldev"),
```

Comment:
The assert functions had their classes more specific to their assertions.  how come this one is `assert-admiraldev`.  Could it be `assert-filter_cond` to be consistent with the others?

Reply:
Nevermind!! I see my misunderstanding!!